.test-infra/jenkins/LoadTestsBuilder.groovy

```diff
@@ -37,7 +37,22 @@ class LoadTestsBuilder {
         }
     }
 
-    private static String parseOptions(Map<String, Object> options) {
-        options.collect { "--${it.key}=${it.value.toString()}".replace('\"', '\\"') }.join(' ')
+    static void loadTestPython(context, String title, Runner runner, Map<String, ?> options, String mainClass) {
```

Comment:
We could extend the `loadTest` method with `SDK` parameter and choose the configuration based on that there. Same as above lets do this only if it's a trivial refactoring - otherwise we should create a separate pr

Reply:
I agree, with SDK enum it would be simplified